reference/5.1/Microsoft.PowerShell.Utility/Compare-Object.md

```diff
@@ -216,6 +222,39 @@ output displayed by the default format for **System.Boolean** objects didn't dis
 **SideIndicator** property. However, the returned **System.Boolean** object has the added
 **NoteProperty**.
 
+### Example 6 - Comparing complex objects
+
+This example show the behavior when comparing complex objects. In this example we store two
+different process objects for different instances of PowerShell. Both variables contain process
+object with the same name. When the objects are compared without specifying the **Property**
```

Comment:
`objects` should be plural or changed to 'contain a process object'.

Reply:
Fixed